package-overrides/github/angular-ui/ui-layout@1.1.0.json

```diff
@@ -0,0 +1,15 @@
+{
+  "format": "global",
+  "dependencies": {
+    "angular": "^1"
+  },
+  "registry": "jspm",
+  "main":
+    ["./src/ui-layout.js"],
```

Comment:
jspm doesn't stricly support a main array, may be best to make this a plain string.